public/ACMW/css/events.css

```diff
@@ -40,83 +31,62 @@
 .navbar {
 	font-family: Montserrat, sans-serif;
 	margin-bottom: 0;
-	background-color: #2d2d30;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of !important](https://app.codacy.com/manual/AravindVNair99/FACE-Amrita-Bengaluru/pullRequest?prid=5264856)